### PR TITLE
Remove web server functionality

### LINKED ISSUE #4: Remove the "serve" functionality

### PR DESCRIPTION
**Pull request by connorkuehl**
Closes #4.

Signed-off-by: Connor Kuehl <ckuehl@redhat.com>

### CODE REVIEW

**Review by npmccallum — CHANGES_REQUESTED** [review]
This PR removes only the server. It doesn't remove the client functionality for that server.

**Review by lkatalin — DISMISSED** [review]

File: src/main.rs [hunk 1]
```diff
@@ -484,145 +454,14 @@ mod generate {
...
-            exit(1);
-        } else {
-            firmware()
-                .pek_generate()
-                .unwrap_or_exit("unable to rotate OCA, PEK and PDH");
-        }
-
-        exit(0)
+    pub fn cmd(_: &ArgMatches) -> ! {
+        pdh()
```
[the remark this answers is not in the document]
Comment (connorkuehl) [reply src/main.rs:461]: Would you consider this a blocker for an alpha release?

[the remark this answers is not in the document]
Comment (npmccallum) [reply src/main.rs:461]: No.

**Comment by npmccallum** [comment]
@connorkuehl Please rebase.

**Comment by connorkuehl** [comment]
Rebased

**Review by npmccallum — APPROVED** [review]

**Review by mbestavros — APPROVED** [review]

**Pull request merged**